index.html

```diff
@@ -43,5 +44,198 @@ <h3 class="headline3">LET'S CONNECT</h3>
+            </div>
+            <div class="card-works4">
+                <div class="snapshoot-portfolio">
+                    <img src="icons-images/Snapshoot4.png" alt="" />
+                </div>
+                <div class="left-block-container">
+                    <div class="left-block">
+                        <div class="project1">
+                            <h1>Multi-Post </br>Stories</h1>
+                            <ul>
+                                <li class="canopy">CANOPY</li>
+                                <li><img src="icons-images/Counter.png" /></li>
+                                <li class="role">Back End Dev</li>
+                                <li><img src="icons-images/Counter.png" /></li>
+                                <li class="role">2015</li>
+                            </ul>
+                        </div>
+                        <div class="project-content">
+                            <p>
+                                A daily selection of privately personalized reads; 
+                                no accounts or sign-ups required.
+                            </p>
+                        </div>
+                        <div class="project-list">
+                            <ul>
+                                <li>html</li>
+                                <li>css</li>
+                                <li>javascript</li>
+                            </ul>
+                        </div> 
+                        <a href="#" class="button">
+                            See Project
+                        </a>
+                    </div>
+                </div>
+            </div>
+            <div class="about">
+                <div class="about1d">
+                    <h1 class="about1">
+                        About <br> Myself
+                    </h1>
+                </div>
+                <div class="about2d">
+                    <p class="about2">
+                        Hello I’m a software developer! I can help you build a product, 
+                        feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                    </p>
+                </div>
+                <div class="about3d">
+                    <h3 class="about3">LET'S CONNECT</h3>
+                </div>
+                <div class="social-media">
+                    <ul class="social-media-icon2">
+                        <li><a href="#"><img src="/icons-images/Vector (2).png"></a></li>
+                        <li><a href="#"><img src="/icons-images/Linkedin icon.png"></a></li>
+                        <li><a href="#"><img src="/icons-images/Vector (3).png"></a></li>
+                        <li><a href="#"><img src="/icons-images/Vector.png"></a></li>
+                        <li><a href="#"><img src="/icons-images/Vector (1).png"></a></li>   
+                    </ul>
+                </div>
```

Comment:
- [ ] Hi @Techcolony, nice working on your finished mobile version of the portfolio project, currently, the social media icon links are broken in both the header section and the about section, and also the Hamburger icon link is broken as well. Kindly check the image directory and be sure you are targeting them perfectly,
  - Secondly, the background image in the header section is not showing kindly fix that too.
  - The icon in front of the Language should be spaced to the right and aligned perfectly just as you did for the icons in front of the Framework and Skills


| Header  | About |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/78634841/221379319-e3236a16-5fb1-4352-a689-baf12d116a64.png) | ![saheed-1](https://user-images.githubusercontent.com/78634841/221379348-e92187dc-2ad5-4a88-b616-1c978ba54b9d.png) |